internal/tenant/internal/enforcement/enforcement.go

```diff
@@ -1,11 +1,24 @@
 package enforcement
 
-import "go.uber.org/atomic"
+import (
+	"os"
+	"runtime/pprof"
+
+	"go.uber.org/atomic"
+)
 
 // EnforcementMode is the current tenant enforcement mode. It resides here
 // instead of in the tenant package to avoid a circular dependency. See
 // tenanttest.MockEnforce.
-var EnforcementMode atomic.String
+var EnforcementMode = atomic.NewString(os.Getenv("SRC_TENANT_ENFORCEMENT_MODE"))
+
+var PPROFUniqID atomic.Int64
+var PPROFMissingTenant = func() *pprof.Profile {
```

Comment:
this doesn't feel like it should live in here and be exported? The only placed you use it is in internal/tenant

Also why not wrap it in a sync.OnceValue to avoid registring if zoekt is just imported as a library and not run?

Reply:
wrapped it in a sync.Once and moved it to internal/tenant. I thought I had circular imports but apparently I do not ;-)